include/swift/AST/DiagnosticsDriver.def

```diff
@@ -109,7 +109,7 @@ WARNING(warn_cannot_stat_input,none,
         (StringRef, StringRef))
 
 WARNING(warn_unable_to_load_dependencies, none,
-        "unable to load dependencies file \"%0\", disabling incremental mode",
+        "unable to load dependencies file '%0', disabling incremental mode",
```

Comment:
I think this is a file name too; not sure about the change above it but it also refers to a file map. The rest of the changes lgtm!

Reply:
I  think the one above it returns the `NAME` of one of these: https://github.com/apple/swift/blob/9370929518b0053e61f2bf94754410841091055e/include/swift/Basic/FileTypes.def, which is not a file name

I'll remove the highlighted change.

Reply:
done in cb507c514da88d7c10ed1d725a32a36bf831b619